src/sentry/integrations/bitbucket/integration.py

```diff
@@ -171,11 +171,14 @@ def build_integration(self, state):
             principal_data = state["principal"]
 
             domain = principal_data["links"]["html"]["href"].replace("https://", "").rstrip("/")
+            if "nickname" in principal_data:
```

Comment:
Can you add a comment saying why we need to modify the domain and link the bitbucket documentation?